### ISSUE TITLE
Could not detect a platform family for Linux opensuse-leap distribution

### ISSUE THREAD

**Issue by m-terel** [shirou/gopsutil#1048]
**Describe the bug**
We've found using `gopsutil` quite useful in open-source [rport](https://github.com/cloudradar-monitoring/rport) project. But we've  encountered a problem of detecting a platform family for Linux `opensuse-leap` distribution. Here is an example of `os-release` file:
```
    NAME="openSUSE Leap"
    VERSION="15.2"
    ID="opensuse-leap"
    ID_LIKE="suse opensuse"
    VERSION_ID="15.2"
    PRETTY_NAME="openSUSE Leap 15.2"
    ANSI_COLOR="0;32"
    CPE_NAME="cpe:/o:opensuse:leap:15.2"
    BUG_REPORT_URL="https://bugs.opensuse.org"
    HOME_URL="https://www.opensuse.org/"
```
`PlatformInformationWithContext` detects platform family by `ID` attribute. Since `"opensuse-leap"` is missing in the switch statement, it returns an empty string for `family`.

**Expected behavior**
`PlatformInformationWithContext` should return `suse` value as `family`variable. 

**Environment:**
`Linux HV3-Rport-Client-SuSE-01 5.3.18-lp152.60-default #1 SMP Tue Jan 12 23:10:31 UTC 2021 (9898712) x86_64 x86_64 x86_64 GNU/Linux`

**Additional context**
According to https://en.opensuse.org/SDB:SUSE_and_openSUSE_Products_Version_Outputs there are other distributions that are not supported by `PlatformInformationWithContext`: `"opensuse-tumbleweed", "opensuse-tumbleweed-kubic", "sled", "caasp"`.

BTW, if you like our project https://github.com/cloudradar-monitoring/rport - please give us a star🙂

**Comment by m-terel** [comment]
#1047 fixes this issue.

**Comment by Lomanic** [comment]
Fixed by #1047
